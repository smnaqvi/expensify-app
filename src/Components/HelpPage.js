import React from 'react';

const HelpPage = () => (
    <div>
      This is from the HelpPage
    </div>
  );

  export default HelpPage;