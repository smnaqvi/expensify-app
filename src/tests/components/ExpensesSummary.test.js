import React from 'react';
import {shallow} from 'enzyme';
import { ExpensesSummary } from "../../Components/ExpensesSummary";

test('should render summary correctly with 1 expense', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={1955} />)
    expect(wrapper).toMatchSnapshot();
})

test('should render summary correctly with multiple expenses', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={2} expensesTotal={19522} />)
    expect(wrapper).toMatchSnapshot();
})