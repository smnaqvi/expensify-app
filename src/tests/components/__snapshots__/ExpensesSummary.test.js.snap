// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render summary correctly with 1 expense 1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $19.55
  </h1>
</div>
`;

exports[`should render summary correctly with multiple expenses 1`] = `
<div>
  <h1>
    Viewing 
    2
     
    expenses
     totalling 
    $195.22
  </h1>
</div>
`;
